tests/Model/Generated/Endpoint/PaymentTest.php

```diff
@@ -52,27 +48,33 @@ public function testSendMoneyToOtherUser()
     {
         $this->skipTestIfNeededDueToInsufficientBalance();
 
-        Payment::create(
+        $response = Payment::create(
             new Amount(self::PAYMENT_AMOUNT_IN_EUR, self::PAYMENT_CURRENCY),
             $this->getPointerUserBravo(),
             self::PAYMENT_DESCRIPTION
         );
+
+        static::assertNotNull($response);
     }
 
     /**
      * Test sending money to other monetaryAccount.
      *
      * This test has no assertion as of its testing to see if the code runs without errors.
```

Comment:
This test has no assertion as it is testing whether the code runs without errors.